src/Common/src/System/Diagnostics/Debug.cs

```diff
@@ -8,7 +8,7 @@ namespace System.Diagnostics
     /// <summary>
     /// Provides a set of properties and methods for debugging code.
     /// </summary>
-    static partial class Debug
+    public static partial class Debug
```

Comment:
Will do. I now see why the tool added `public' because the other part of the class is marked public.